views/profile.ejs

```diff
@@ -98,49 +98,17 @@
             </div>
 
         </div>
+    </div>
 
 </body>
 
-<!-- Modal -->
-<div class="modal fade" id="Modal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel"
```

Comment:
@dylanperera ,

This is an important part of the profile page. Why has this section been removed?

Note: It helps bring up the modal for changing your profile picture.